functions/idvtoauth0/main.py

```diff
@@ -103,23 +108,29 @@ def handle(event, context):
                     for g in upstream_user['groups']:
                         if g not in profile['groups']:
                             profile['groups'].append(g)
-                            logger.info("Forced re-integration of LDAP group {} for user {}".format(g, user_id))
+                            logger.info('Forced re-integration of LDAP group: {} for user: {}'.format(g, user_id))
 
                 # Update groups only in Auth0
                 profile_groups = {'groups': profile.get('groups')}
                 res = client.update_user(user_id, profile_groups)
-                logger.info("Updating user group information in auth0 for {user_id}".format(user_id=user_id))
+                logger.info('Updating user group information in auth0 for {}'.format(user_id))
+                logger.debug('-------------------Post-Integration--------------------------')
+                logger.debug(profile)
+                logger.debug('------------------------End----------------------------------')
             except Exception as e:
                 """Temporarily patch around raising inside loop until authzero.py can become part of CIS core."""
                 res = e
-            logger.info("Status of message processing is {s}".format(s=res))
+            logger.info('Auth0 processing complete for for user: {}'.format(res, user_id))
+            logger.debug('-------------------Auth0-Response-----------------------------')
+            logger.debug(res)
+            logger.debug('------------------------End----------------------------------')
         else:
             logger.critical(
-                "User could not be matched in vault for userid : {user_id}".format(user_id=user_id)
+                'User could not be matched in vault for userid : {}'.format(user_id)
             )
 
     logger.info(
-        'Successfully processed {} records.'.format(len(event['Records']))
+        'IDVTOAUTH0: Successfully processed {} records.'.format(len(event['Records']))
```

Comment:
note, this looks like a replacement for the logger name. dont know if that matters (the pattern is repeated everywhere in the code)